exercises/rotational-cipher/description.md

```diff
@@ -0,0 +1,37 @@
+# Rotational Cipher
+
+Create an implementation of the rotational cipher, also sometimes called the Caesar cipher.
+
+The Caesar cipher is a simple shift cipher that relies on
+transposing all the letters in the alphabet using an integer key
+between `0` and `26`. Using a key of `0` or `26` will always yield
+the same output due to modular arithmetic. The letter is shifted
+for as many values as the value of the key.
+
+The general notation for rotational ciphers is `ROT + <key>`.
+The most commonly used rotational cipher is `ROT13`.
+
+A `ROT13` on the Latin alphabet would be as follows:
+
+```plain
+Plain:  abcdefghijklmnopqrstuvwxyz
+Cipher: nopqrstuvwxyzabcdefghijklm
+```
+
+It is stronger than the Atbash cipher because it has 27 possible keys, and 25 usable keys.
+
+Ciphertext is written out in the same formatting as the input including spaces and punctuation.
+
+## Examples
+- ROT5  `omg` gives `trl`
+- ROT0  `c` gives `c`
+- ROT26 `Cool` gives `Cool`
+- ROT13 `The quick brown fox jumps over the lazy dog.` gives `Gur dhvpx oebja sbk whzcf bire gur ynml qbt.`
+- ROT13 `Gur dhvpx oebja sbk whzcf bire gur ynml qbt.` gives `The quick brown fox jumps over the lazy dog.`
+
+## BONUS!
+non-ASCII characters are downgraded this means that an `é` would become an `e`.
```

Comment:
I think, by #428, we wouldn't want to add these. Unless there is some particular reason it's very important to *this* particular problem (but wasn't important to, say, Atbash, whose non-ASCII cases got removed in #529)

Downgrading leads to some interesting questions - now, it's not always the case that `rot13(rot13(t)) = t`. And it leads to conflict on whether `rot26(t) = t` (if downgrading happens, maybe it won't be, unless someone special-cased 0 and 26 to always return the input unchanged, but the nwe're being inconsistent on whether to downgrade)